src/mint/process.rs

```diff
@@ -95,10 +120,24 @@ pub async fn process_mint(args: MintArgs) -> Result<()> {
     };
     println!("\nMinting to {}", &receiver_pubkey);
 
-    let number = args.number.unwrap_or(1);
+    let number = match using_airdrop_list {
+        true => 0,
+        false => args.number.unwrap_or(1),
+    };
+
+    if number > 1 && using_airdrop_list {
```

Comment:
This check will never get triggered.

Reply:
Yeah I was shoe horning the `number` and `airdrop` together, felt like it's getting messy.